src/test/lib/DataTest.cpp

```diff
@@ -14,6 +14,10 @@
 #include "DataTest.cpp.clog.h"
 #endif
 
+#ifdef QUIC_USE_RAW_DATAPATH
+extern bool UseQTIP;
+#endif
```

Comment:
Would it be better to make this bool always present but only ever set to true when RAW is used? That'd simplify all these #ifdefs

Reply:
That can't be shared across user and kernel mode.